src/test/java/com/agnieszkapawska/flashcards/api/controllers/FlashcardControllerTest.java

```diff
@@ -15,18 +15,19 @@
 public class FlashcardControllerTest extends FlashcardsApplicationAbstractTests {
 
     @Test
-    public void shouldReturnStatusOkWhenSavingFlashcard() {
+    public void shouldReturn_StatusOk_WhenSavingFlashcard() {
         //given
         when(super.flashcardFacade.saveFlashcard(any(FlashcardDto.class))).thenReturn(super.flashcardSaveResponseDto);
         //when
         ResponseEntity<FlashcardSaveResponseDto> responseEntity =
                 testRestTemplate.postForEntity(super.baseUrl + "/flashcard", super.flashcardDto, FlashcardSaveResponseDto.class);
         //then
         Assert.assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
+        Assert.assertEquals(super.flashcardSaveResponseDto, responseEntity.getBody());
     }
 
     @Test
-    public void shouldReturnStatusConflictWhenSavingFlashcard() {
+    public void shouldReturn_StatusConflict_WhenSavingFlashcard_When_CouldNotCreateEntity() {
```

Comment:
Too much.
`shouldReturnStatusConflict_WhenSavingFlashcard_WhenCouldNotCreateEntity`.

Or you could even consider changing words order to
`savingFlashcard_ShouldReturnStatusConflict_WhenCouldNotCreateEntity`.
It's more natural IMO.